salt/kafka-tool/init.sls

```diff
@@ -0,0 +1,81 @@
+#read all pillar data
+
+{% set install_dir = pillar['pnda']['homedir'] %}
+{% set packages_server = pillar['packages_server']['base_uri'] %}
+{% set release_directory = salt['pillar.get']('kafkatool:release_dir', '/opt/pnda') %}
+{% set release_version = salt['pillar.get']('kafkatool:version', '0.3.0')  %}
+{% set release_filename = 'kafka-tool-' + release_version + '.tar.gz' %}
+
+{% set p  = salt['pillar.get']('kafka', {}) %}
+{% set local_kafka_path = p.get('prefix', '/opt/pnda/kafka') %}
+{% set p1  = salt['pillar.get']('kafkatool', {}) %}
+{% set local_kafkat_log_path = p1.get('log_dir', '/var/log/pnda/kafkatool') %}
+
+{%- set zk_ips = [] -%}
+{%- for ip in salt['pnda.kafka_zookeepers_ips']() -%}
+{%-   do zk_ips.append(ip + ':2181') -%}
+{%- endfor -%}
+
+
+kafkat-dl-and-extract:
+  archive.extracted:
+    - name: {{ install_dir }}
+    - source: {{ packages_server }}/{{ release_filename }}
+    - source_hash: {{ packages_server }}/{{ release_filename }}.sha512.txt
+    - archive_format: tar
+    - tar_options: v
+    - if_missing: {{ release_directory }}/kafka-tool-{{ release_version }}
+
+kafka-tool-create_link:
+  file.symlink:
+    - name: {{ release_directory }}/kafka-tool
+    - target: {{ release_directory }}/kafka-tool-{{ release_version }}
+
+install ruby:
+  pkg.installed:
+{% if grains['os'] == 'RedHat' %}
```

Comment:
use {{ pillar['ruby-devel']['package-name'] }} which then will use the right pillar/packages depending on the OS as defined in pillar/top.sls